cli/runtime/docs/README.md

```diff
@@ -0,0 +1,34 @@
+# Tanzu CLI Integration Library
+
+This library exposes many different functionalities that plugin developers can use while developing Tanzu CLI plugins. Below are some of the functionalities that are available:
+
+1. Component library
+1. Config library
+1. Plugin helpers
+1. Command helpers
+1. Test helpers
+
+## Component Library
+
+This package implements helper functions for the components listed below:
+
+- output writers (table, listtable, json, yaml, spinner)
+- prompt
+- selector
+- question
+
+## Config Library
+
+This package implements helper functions to read, write and update the tanzu configuration file (`~/.config/tanzu/config.yaml`).
+
+## Plugin Helpers
+
+This package implements helper functions for new plugin creation. This is one of the main packages that each and every plugin will need to import to develop integrate with the Tanzu CLI.
```

Comment:
"develop integrate" : extra "develop"
That was my typo in my original comment. 😬

Reply:
My bad as well :)